### ISSUE TITLE
Consider HTTP 300 as realization for list profiles / list tokens

### ISSUE THREAD

**Issue by azaroth42** [w3c/dx-connegp#5]
OPTIONS is not cacheable, and there isn't a profile parameter for the Link header construct ... so ... how about an HTTP 300 Multiple Choices response?

HTTP 300 says:

> For request methods other than HEAD, the server SHOULD generate a
> payload in the 300 response containing a list of representation
> metadata and URI reference(s) from which the user or user agent can
> choose the one most preferred.

This would allow the response to describe the URIs of the representations, and their metadata, thus including both profile URI and token, if any. The disadvantage is that it would be a custom response format, but one that is likely very obvious and could tie back to other DXWG deliverables.

**Comment by larsgsvensson** [comment]
@azaroth42 We discussed this issue in the CNEG meeting on 2019-09-12. The [decision](https://www.w3.org/2019/09/12-dxwgcneg-minutes#x13) was to move this to future work. Is that OK for you?

**Comment by nicholascar** [comment]
I agree with marking as future work, this comment’s just for future readers.

> it would be a custom response format

Goven that we now have the Alternate Profiles data model, the body payload could be conformant with that, serialised in an RDF format or tabulated in HTML (we have examples of both in the doc).

**Comment by rob-metalinkage** [comment]
possible solution:  after 

The server SHOULD attempt to reply with a profile that best matches the client request. 

add

The server MAY respond with the ALTR representation and a HTTP 300 code as a fallback if unable to determine an appropriate response, such as when an explicit media type not supported is requested.

**Comment by YoucTagh** [comment]
As discussed in the meeting 24/01/2024, the proposed wording is: 

> As specified in [[RFC9110]], If the HEAD method is used in a request that specifies preferences in the profile, media type, or other negotiation dimension, then the response should be the same as for a GET method, including headers and status code, except that the body MUST be empty. It is important to note that headers, for which the value is determined only while generating the response content, MAY be omitted
